packages/webpack-extraction-plugin/package.json

```diff
@@ -17,6 +18,10 @@
   },
   "peerDependencies": {
     "@griffel/core": "^1.5.1",
+    "browserslist": "^4.19.1",
```

Comment:
We don't need it in `peerDependencies` as we have it under `dependencies`, right?